### PR TITLE
feat: prepare v0.4.4 and relax tag publishing rules

### PR DESCRIPTION
**Pull request by unnawut**
## Overview

Bump up `VERSION` in preparation for `v0.4.4` and relax circleci publishing rules for tags

## Changes

- Bump `VERSION` to 0.4.4
- Loosen the CircleCI rules for publishing on tags. This is an attempt to debug why CircleCI is not publishing them. We only push tags for releases so loosen this up shouldn't be an issue.

## Testing

N/A

### CODE REVIEW

**Comment by InoMurko** [comment]
when is the release rolling out?

**Comment by jbunce** [comment]
@InoMurko when I get the image :)

**Comment by InoMurko** [comment]
Aha! OK. 
1. docker-compose-watcher.yml @unnawut please bump the SHA tags
2. Infra needs to set `LOGGER_BACKEND=console` because json logs are not ready on the infra level yet

File: .circleci/config.yml [hunk 1]
```diff
@@ -748,8 +747,7 @@ workflows:
                 - /^v[0-9]+\.[0-9]+/
```
Comment (jrhite) [inline .circleci/config.yml]: is this one left behind intentionally?

**Comment by unnawut** [comment]
We still need to do a load test on this bundle, get all downstream products tested with this version and the actual publishing, so I'm guesstimating we're looking at up coming Monday 2 March the earliest.

File: .circleci/config.yml [hunk 2]
```diff
@@ -725,8 +725,7 @@ workflows:
...
```
Comment (jrhite) [inline .circleci/config.yml]: is this one left behind intentionally?

**Comment by jbunce** [comment]
@InoMurko can we keep this release just for the fix? I know it's tempting to get the new feature in now but we're right in the middle of a refactor for versioning of our Helm charts.

File: .circleci/config.yml (hunk 1 above)
Comment (unnawut) [reply .circleci/config.yml]: Yup. The branches are working fine.

File: .circleci/config.yml (hunk 2 above)
Comment (unnawut) [reply .circleci/config.yml]: Yup. The branches are working fine.

**Comment by unnawut** [comment]
> docker-compose-watcher.yml @unnawut please bump the SHA tags

I keep forgetting this, and probably will keep forgetting this 🤦‍♂ Any idea if we can automate/lint this?

**Pull request merged**